### DISCUSSION TITLE
PvP feedback round 3

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14781] [Unstable feedback]
# General observations:

**1. Need more Respawn gear, for 2 reasons**
After respawning with only a revolver and a knife, you spend a considerable amount of time looting your own sub. You have to pause driving the sub until you grab basic survival gear (some meds, a welder, an oxygen mask...) 
 In outpost deathmatch, you often shoot 6 bullets and then cannot even loot enemy corpses that also ran out of bullets. 
**Add 6 revolver bullets, 1 diving mask, 2 bandages 1 morphine 1 saline to the starting gear.**

**2. Server Lobby needs a team voice channel**
Both teams on the same voice channel, you can't tell you team to pick certain subs or certain perks. 
Local chat key should be Team, Radio chat key should be global

**3. Stun reduction should be 80% by default**
PvP was somewhat unplayable with 0% stun reduction as you'd get stunlocked by most weapons
Or maybe a "stun resistance" affliction associated with stun to prevent stunlocks (stun would have a statuseffect that builds up the stun resistance affliction so each new application of stun would be weaker)
See post below https://github.com/FakeFishGames/Barotrauma/discussions/14781#discussioncomment-10858956

**4. Secure lockers can be ID locked**
Only captain and security can take items from it

**5. Exosuit OP**
It just is, especially in KOTH

--------------------

# King of the Hull

**1. IT'S NOT MADE FOR SUBMARINES**
This is what the average KotH round looks like: a broken sub sitting on top of the beacon, making it inaccessible and staying there for the rest of the round. 

![image](https://github.com/user-attachments/assets/66d5ea49-330b-409a-adfd-feb8c9e558f6)

There's an incentive to play JUST LIKE A REGULAR SUBvSUB GAME and win the "duel" to control the beacon.
It's even worse if the beacon is a ceiling one and a sub manages to dock: the other sub sinks while this one is latched onto the ceiling and stays there.
KOTH is made for shuttles that despawn, not huge subs that are obstructions and blast themselves as well as the beacon

**2. NEED FAST SHUTTLES**
KOTH is however good if you have FAST RESPAWN TIMES and SHUTTLES.
We had to use a custom, fast shuttle with a small coilgun on a 30s respawn timer and the game mode was a blast. None of the current shuttles are fast enough, nor do they have enough loot in them. 

**3. UNBREAKABLE BEACON WALLS**
Some beacons have unbreakable walls. People kept shooting at beacons or trying to plasma cut into them, thinking there was a bug.
Unbreakable walls arent intuitive in PvE and certainly not in PvP (the beacon crew is expected to have welders to prevent break-ins).

![image](https://github.com/user-attachments/assets/a2246069-0d73-49ef-a24e-25fceb9fa4c0)

---------------------

# Outpost Deathmatch

**1. Need more equipment, more often.**
We played with maybe 10 players and we were constantly STARVING for equipment for the first half of the deathmatch. This goes hand in hand with the lack of starter gear. 
Crates were too scarce

**2. Pink "ramps" can be broken**
They're not invulnerable and you can get stuck if broken?
![image](https://github.com/user-attachments/assets/36648ddc-69b8-4e5e-8dca-a704e879b848)

**3. Walls should be invulnerable**
It honestly breaks the game if you start flooding the outpost, and small leaks caused constant annoying water noises and particles. Turrets shooting explosive ammo can sometimes break walls as well.

**4. Layout improvements (RaateColony)**

- The sewer is useless/bait, it leads to 2 turrets as soon as you open the hatch.
Turrets in the fabrication room should be moved back so you can fight there (and use sewer exit)
![image](https://github.com/user-attachments/assets/25a03660-1bdf-4110-86e4-e3993879a089)

- Spawn "flow" isnt intuitive, sometimes it spawns you into a random spot and you have to zigzag across random rooms. 
Simplify the flow to make it less confusing
![image](https://github.com/user-attachments/assets/815db077-2f38-4983-930d-eda645cf2f8b)
![image](https://github.com/user-attachments/assets/0d28aea6-d566-4e45-a524-f32546353654)

- More "depression" areas using platforms
On flat parts of the outpost, add depressions with a platform which allows players to go lower so other players can easily shoot above them. Since Baro has no Z depth, adding soft elevation matters
![image](https://github.com/user-attachments/assets/90c4f449-31df-4bfe-80cf-6965ac5000ab)

**Comment by ubertpendragon** [comment]
Attempt at making a self building stun resistance for microstuns:
[Stun resistance.zip](https://github.com/user-attachments/files/17269862/Stun.resistance.zip)

**Comment by ubertpendragon** [comment]
Flamers kind of ruin the round as the entire outpost is on fire

![image](https://github.com/user-attachments/assets/04b218d8-befc-4b6a-ab66-0a281f88d4de)

**Comment by Tsunder** [comment]
1. Levels all being 700-1000m make for boring same sized maps. Longer maps are nice for variety and allow for more strategic approaches.
2. Pre-game points usage is quite unbalanced, for example rifles and exosuits are quite desired.
3. Pre-game points usage seems dubious and I'm not sure it's a workable at all.
4. default/autofill items or lackthereof 
5. would really like more host customizations, like blocking items

While I like the work, I'm concerned for the quality of matches when people are more competitive.

**Comment by Crazyjr20** [comment]
In Outpost Deathmatch the crates can often be too slow to keep the supply for a lobby worth of players, meaning you spend more time shooting a pea shooter that has half the bullets dissapear due to lag and then just the knife rather than any actual weapons.
The simplest thing for this would be just reducing the wait timer between crate spawns.
An alternative approach could actually be having a unique spawnpoint within team spawns for some kind of weapon drop that upgrades over time but a tier lower than the ones around the map, so people can continue to actually fight more in the mode for fighting.

**Reply by girlyguppy** [reply]
or even better- give the mapmaker or the host control of the spawns

**Reply by girlyguppy** [reply]
(me, im the mapmaker/host)
